pkg/pipelinectxt/artifacts.go

```diff
@@ -27,6 +28,12 @@ const (
 	SonarAnalysisPath         = ArtifactsPath + "/" + SonarAnalysisDir
 	AquaScansDir              = "aquasec-scans"
 	AquaScansPath             = ArtifactsPath + "/" + AquaScansDir
+	TrivyScansDir             = "trivy-scans"
+	TrivyScansPath            = ArtifactsPath + "/" + TrivyScansDir
+	SbomsDir                  = "sboms"
+	SbomsPath                 = ArtifactsPath + "/" + SbomsDir
+	SbomsFormat               = "spdx"
+	SbomsFilename             = "sbom.spdx"
```

Comment:
I would suggest:

```go
SBOMsDir                  = "sboms"
SBOMsPath                 = ArtifactsPath + "/" + SbomsDir
SBOMFormat               = "spdx"
```

I think we can't define `SbomsFilename` here because that needs to depend on the image name (to avoid overwriting the file when two `ods-package-image` tasks are chained).

Reply:
we were also discussing why using plural since it is always just a file and that folder will only contain one file, should we also change that? might also apply to other artifacts

Reply:
No I would like to keep the plural because there actually may be multiple files, e.g. when multiple `ods-package-image` tasks are chained. Also in the future, we may want to save multiple formats.